modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/EpicsUtil.scala

```diff
@@ -17,13 +17,17 @@ trait EpicsCommand {
 
   protected val cs: Option[CaCommandSender]
 
-  def post: SeqAction[Unit] = safe(EitherT(Task.async[TrySeq[Unit]](f => cs.map(_.postCallback {
-    new CaCommandListener {
-      override def onSuccess(): Unit = f(TrySeq(()).right)
-
-      override def onFailure(cause: Exception): Unit = f(cause.left)
+  def post: SeqAction[Unit] = safe(EitherT(
+    Task.async[TrySeq[Unit]](f => {
+      cs.foreach(_.postCallback {
+        new CaCommandListener {
+          override def onSuccess(): Unit = f(TrySeq(()).right)
+
+          override def onFailure(cause: Exception): Unit = f(cause.left)
+        }
+      } )
+      ()
```

Comment:
This one baffled me. I guess we don't care about the result of the `async` call?